### PR TITLE
Added data test id for draft table & draft filter

### PR DESCRIPTION
**Pull request by auslin-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE
https://aottech.atlassian.net/browse/FWF-2853

# Changes
Added data test ID for draft table & draft filter

### CODE REVIEW

**Review by shuhaib-aot — COMMENTED** [review]
Looks good but filter click button's data-testid missed:
image
you can see the filter button there. if we need to open the filter  should data-testid need to be there

**Review by abilpraju-aot — APPROVED** [review]

**Comment by auslin-aot** [comment]
> Looks good but filter click button's data-testid missed: image you can see the filter button there. if we need to open the filter should data-testid need to be there

Done

File: forms-flow-web/src/components/Draft/DraftFilter.js [hunk 1]
```diff
@@ -148,13 +152,15 @@ const DraftFilter = ({ setDisplayFilter, filterParams, setFilterParams }) => {
...
             onClick={closeFilterModal}
+            data-testid="draft-filter-close-button"
           >
             {t("Cancel")}
           </button>
           <button
             className="btn btn-dark"
             onClick={applyFilters}
             disabled={!isSearchParamEntered}
+            data-testid="draft-filter-apply-filter-button"
```
Comment (fahad-aot) [inline forms-flow-web/src/components/Draft/DraftFilter.js:163]: "draft-filter-apply-button"  is enough , right?

Comment (auslin-aot) [reply forms-flow-web/src/components/Draft/DraftFilter.js]: done

**Review by fahad-aot — APPROVED** [review]

**Review by shuhaib-aot — APPROVED** [review]

**Pull request merged**